<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectBuildEnvironmentDialog</class>
 <widget class="QDialog" name="ProjectBuildEnvironmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Project Environment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="envTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="toolTip">
        <string>Add Environment Variable</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/inc_add.png</normaloff>:/images/inc_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="toolTip">
        <string>Delete Environment Variable</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/inc_del.png</normaloff>:/images/inc_del.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearPushButton">
       <property name="toolTip">
        <string>Remove All Environment Variables</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/env_clear.png</normaloff>:/images/env_clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useSystemPushButton">
       <property name="toolTip">
        <string>Set System Environment Variables</string>
       </property>
       <property name="text">
        <string>Use System</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/env_system.png</normaloff>:/images/env_system.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importPushButton">
       <property name="toolTip">
        <string>Import Environment Variables From Other Project</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset resource="asproject.qrc">
         <normaloff>:/images/env_import.png</normaloff>:/images/env_import.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="asproject.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectBuildEnvironmentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectBuildEnvironmentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
